packages/p2p/src/components/my-ads/form-ads.jsx

```diff
@@ -255,6 +263,18 @@ const FormAds = observer(() => {
                     );
                 }}
             </Formik>
+            <Modal
+                className='p2p-my-ads__modal-error'
+                is_open={is_api_error_modal_visible}
+                small
+                has_close_icon={false}
+                title={localize('Something’s not right')}
+            >
+                <Modal.Body>{my_ads_store.api_error_message}</Modal.Body>
```

Comment:
Should it be wrapped in a `<Text>` component? Make sure to check on dark + light theme.